cmd/pj-rehearse/main.go

```diff
@@ -86,6 +86,7 @@ an infrastructure issue.`
 
 pj-rehearse rehearsed jobs and at least one of them failed. This means that
 job would fail when executed against the current HEAD of the target branch.`
+	REHEARSAL_THRESHOLD = 15
```

Comment:
Go prefers `rehearsalThreshold`

Reply:
Aaaargh capitalized constants are burned to my brain